meta/no_message.yaml

```diff
@@ -0,0 +1,15 @@
+rules:
```

Comment:
Same, rename empty_message.test.yaml

Reply:
I have a different test called empty_message.yaml, which checks if you have a message field but the text is "". Should I merge these two? Or do you have a name suggestion for that one?

Reply:
I'm not sure I understand what you mean. In this PR there is a YAML rule, empty_message.yaml that detects empty messages, and a YAML test file, no_message.yaml. I just want you to rename no_message.yaml to empty_message.test.yaml (with the .test.yaml extension) to better see what is a rule and what is a test file. Otherwise with semgrep rules about semgrep rules it can be quite confusing.